bebop_driver/src/bebop_driver_nodelet.cpp

```diff
@@ -392,6 +412,19 @@ void BebopDriverNodelet::TakeSnapshotCallback(const std_msgs::EmptyConstPtr &emp
   }
 }
 
+void BebopDriverNodelet::SetPictureFormatCallback(const std_msgs::UInt8ConstPtr& format_ptr)
+{
+  try
+  {
+    ROS_INFO("Setting picture format to %f", format_ptr->data);
```

Comment:
I don't think %f is correct here, since the input is of type `uint8_t`. Please fix.